server/controllers/user.controllers.js

```diff
@@ -158,6 +159,18 @@ const UserController = {
       })
       .catch((error) => next(error));
   },
+
+  getAccountOverview(req, res, next) {
+    const userId = req.user._id;
+    getAccountOverview(userId)
+      .then((overview) => {
+        res.status(httpStatus.OK).json({
+          success: true,
+          overview,
```

Comment:
I prefer we should be more explicit to make the endpoint easier to work with.
```suggestion
          accountOverview,
```